integration-tests/tests/api/audit-logs/audit-log-record.spec.ts

```diff
@@ -0,0 +1,199 @@
+});
+
+const auditLogArray = graphql(`
+  query GetAllAuditLogsArray($selector: OrganizationSelectorInput!) {
+    organization(selector: $selector) {
+      organization {
+        id
+        slug
+        auditLogs {
+          edges {
+            node {
+              __typename
+            }
+          }
+        }
+      }
+    }
+  }
+`);
+
+test.concurrent('Create multiple Audit Log Records for Organization', async ({ expect }) => {
+  const { ownerToken, createOrg } = await initSeed().createOwner();
+  await waitFor(5000);
+  const { organization, createProject } = await createOrg();
+  await waitFor(5000);
+  await createProject(ProjectType.Single);
+  await waitFor(4000);
+
+  const result = await execute({
+    document: auditLogArray,
+    variables: {
+      selector: {
+        organizationSlug: organization.slug,
+      },
+    },
+    authToken: ownerToken,
+  });
+
+  const auditLogs = result.rawBody.data?.organization?.organization.auditLogs.edges;
+
+  expect(auditLogs?.length).toBe(5);
+  expect(auditLogs?.length).not.toBe(0);
+  expect(
+    auditLogs?.find((log: any) => log.node.__typename === 'OrganizationCreatedAuditLog'),
+  ).toBeDefined();
+  expect(
+    auditLogs?.find((log: any) => log.node.__typename === 'ProjectCreatedAuditLog'),
+  ).toBeDefined();
+  expect(
+    auditLogs?.find((log: any) => log.node.__typename === 'TargetCreatedAuditLog'),
+  ).toBeDefined();
+});
+
+const GetAuditLogs = graphql(`
+  query GetAllAuditLogs($selector: OrganizationSelectorInput!) {
+    organization(selector: $selector) {
+      organization {
+        id
+        slug
+        auditLogs {
+          edges {
+            node {
+              id
+              __typename
+              eventTime
+            }
+          }
+        }
+      }
+    }
+  }
+`);
+
+test.concurrent('Create Audit Log Record for Organization', async ({ expect }) => {
+  await waitFor(5000);
+  const { createOrg, ownerToken } = await initSeed().createOwner();
+  await waitFor(5000);
+  const firstOrg = await createOrg();
+  waitFor(15000);
+  const result = await execute({
+    document: GetAuditLogs,
+    variables: {
+      selector: {
+        organizationSlug: firstOrg.organization.slug,
+      },
+    },
+    authToken: ownerToken,
+  });
+  const auditLogs = result.rawBody.data?.organization?.organization.auditLogs.edges;
+  waitFor(4000);
+  expect(auditLogs?.[0].node.__typename === 'OrganizationCreatedAuditLog');
+  expect(auditLogs?.[0].node.eventTime).toBeDefined();
+});
+
+const ExportAllAuditLogs = graphql(`
+  mutation exportAllAuditLogs($selector: OrganizationSelectorInput!, $filter: AuditLogFilter!) {
+    exportOrganizationAuditLog(selector: $selector, filter: $filter) {
+      ok {
+        url
+      }
+      error {
+        message
+      }
+      __typename
+    }
+  }
+`);
+
+const today = endOfDay(new Date());
+const lastYear = startOfDay(new Date(new Date().setFullYear(new Date().getFullYear() - 1)));
+
+test.concurrent(
```

Comment:
TODO @TuvalSimha please apply the same feedback that I mentioned above to these test cases as well.